### PR TITLE
Rapid7 InsightVM Ansible Role

### PR DESCRIPTION
**Pull request by thomty**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A SRE team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a SRE ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/spaces/SRE/pages/28967861/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).
  - [ ] Think about how this change will affect Open edX operators.  Have you updated the wiki page for the next Open edX release?

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @thomty!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


This is currently a draft pull request. When it is ready for our review and all tests are green, click "Ready for Review", or remove "WIP" from the title, as appropriate.




:warning: We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](https://openedx.org/cla)     or indicated your institutional affiliation. Please see the [CONTRIBUTING](https://github.com/openedx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.     If you've signed an agreement in the past, you may need to re-sign.  See     [The New Home of the Open edX Codebase](https://open.edx.org/blog/the-new-home-of-the-open-edx-codebase/)     for details.

    
Once you've signed the CLA, please allow 1 business day for it to be processed.  After this time, you can re-run the     CLA check by editing the PR title.  If the problem persists, you can tag the `@openedx/cla-problems` team in a     comment on your PR for further assistance.

File: playbooks/roles/rapid7/crowdstrike_falcon.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
```
Comment (adzuci) [inline playbooks/roles/rapid7/crowdstrike_falcon.yml]: I think this file should be in a separate crowdstrike role folder.

File: playbooks/roles/insightvm-agent/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+InsightVM Agent Ansible Role
```
Comment (adzuci) [inline playbooks/roles/insightvm-agent/README.md:1]: @nadeemshahzad Do you know if we have a style guide that specifies if role names should have hyphens or dashes?

**Comment by adzuci** [comment]
Closing as https://github.com/openedx/configuration/pull/6904 replaces this PR.

**Comment by openedx-webhooks** [comment]
@thomty Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/configuration&entry.867055334=https://github.com/openedx/configuration/pull/6900&entry.1484655318=https://github.com/thomty&entry.752974735=2023-03-24+13:20&entry.1917517419=2023-04-03+15:47&entry.2133058324=No) so we can improve your experience in the future.

**Pull request closed without merging**